x-pack/plugins/observability/public/pages/rules/components/status_context.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useCallback, useMemo } from 'react';
+import {
+  EuiPopover,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import { Status } from './status';
+import { RuleStatus, StatusContextProps } from '../types';
+import { statusMap } from '../config';
+
+export function StatusContext({
+  item,
+  onStatusChanged,
+  enableRule,
+  disableRule,
+  muteRule,
+}: StatusContextProps) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const [isUpdating, setIsUpdating] = useState(false);
+  const togglePopover = useCallback(() => setIsPopoverOpen(!isPopoverOpen), [isPopoverOpen]);
+
+  const currentStatus = item.enabled ? RuleStatus.enabled : RuleStatus.disabled;
+  const popOverButton = useMemo(
+    () => <Status type={currentStatus} onClick={togglePopover} />,
+    [currentStatus, togglePopover]
+  );
+
+  const onContextMenuItemClick = useCallback(
+    async (status: RuleStatus) => {
+      togglePopover();
```

Comment:
The popover disappears ONLY when enabled/disabled is clicked. I'm expecting clicking outside will make the popover disappear. 
<img width="519" alt="Screenshot 2022-03-15 at 16 15 22" src="https://user-images.githubusercontent.com/6838659/158568799-c70b087a-9e1d-4ae1-86a2-253dae5e10de.png">

Reply:
@fkanout Yep good observation, I've noticed that as well. I fixed it now and popover disappears when clicking outside as well.